web/src/main/webapp/resources/js/custom.js

```diff
@@ -94,6 +96,49 @@ function sendPost(question) {
 }
 
 function plotLineplot(payloadObj) {
+
+    if (linePlot != null) {
+        linePlot.destroy();
+    }
+
+    dataObject = JSON.parse(payloadObj.data);
+
+    datasetsObjectFormatted = [];
+
+    //create color array to rotate through
```

Comment:
Can you say something here about the colors that we're rotating through - a pointer to the color scheme or comments for each color?